<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Brute Force OTG</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>376</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalls">
    <property name="title">
     <string>Calls</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuCalls"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionConfigure"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
   <addaction name="actionOpenDict"/>
   <addaction name="actionWakePhone"/>
   <addaction name="actionPhotoSensor"/>
   <addaction name="separator"/>
   <addaction name="actionStartBF"/>
   <addaction name="actionStopBF"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About program</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;onnect</string>
   </property>
   <property name="toolTip">
    <string>Connect to serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure</string>
   </property>
   <property name="toolTip">
    <string>Configure serial port</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;lear</string>
   </property>
   <property name="toolTip">
    <string>Clear data</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpenDict">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/wordlist.ico</normaloff>:/images/wordlist.ico</iconset>
   </property>
   <property name="text">
    <string>Open Dict</string>
   </property>
   <property name="toolTip">
    <string>Open Dictionary</string>
   </property>
  </action>
  <action name="actionWakePhone">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/phone.ico</normaloff>:/images/phone.ico</iconset>
   </property>
   <property name="text">
    <string>Wake up phone</string>
   </property>
  </action>
  <action name="actionPhotoSensor">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/lightbulb.ico</normaloff>:/images/lightbulb.ico</iconset>
   </property>
   <property name="text">
    <string>Photosensor</string>
   </property>
   <property name="toolTip">
    <string>Get photo sensor</string>
   </property>
  </action>
  <action name="actionStartBF">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/play.ico</normaloff>:/images/play.ico</iconset>
   </property>
   <property name="text">
    <string>Start brute-force</string>
   </property>
  </action>
  <action name="actionStopBF">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/stop.ico</normaloff>:/images/stop.ico</iconset>
   </property>
   <property name="text">
    <string>Stop brute-force</string>
   </property>
  </action>
  <action name="actionResume">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/inarrow.ico</normaloff>:/images/inarrow.ico</iconset>
   </property>
   <property name="text">
    <string>Resume</string>
   </property>
   <property name="toolTip">
    <string>Resume</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="terminal.qrc"/>
 </resources>
 <connections/>
</ui>
